packages/prover/src/cli/cmds/start/options.ts

```diff
@@ -53,15 +53,15 @@ export const startOptions: CliCommandOptions<StartArgs> = {
   },
 
   beaconUrls: {
-    description: "The beacon node PRC urls for 'rest' mode.",
+    description: "Urls of the beacon nodes to connect to.",
     type: "string",
     array: true,
     conflicts: ["beaconBootnodes"],
     group: "beacon",
   },
 
   beaconBootnodes: {
-    description: "The beacon node PRC urls for 'p2p' mode.",
+    description: "Urls of beacon bootnodes to connect to.",
```

Comment:
I am a bit confused what is meant by `beaconBootnodes`, on the one hand in the tests is passes a URL
https://github.com/ChainSafe/lodestar/blob/42e883586f3bbea1867b8b9c2458ed4b97216480/packages/prover/test/e2e/cli/cmds/start.test.ts#L33-L34

But if those are bootnodes, you would not pass a URL here but an ENR and not connect to them but use them for peer discovery.

This option also give users a false assumption that you can run the light client in p2p mode but this is not  even implemented.

Reply:
Right, I agree it's confusing.
My preference would be:

* do not touch the `beaconBootnodes` as part of this PR
* remove the `beaconBootnodes` CLI option in a follow up PR

It's pretty easy to restore this option once it's actually supporte. It also reduces potential user confusion that might see this option exist but not clearly understand that it's not supported because not yet implemented.

I let @nazarhussain shim in if he feels like it should be kept.

Reply:
I'd agree, let's remove any changes to beaconBootnodes from this PR and remove it separately to have more visibility on it